### ISSUE TITLE
Locks up upon reaching "authorization"

### ISSUE THREAD

**Issue by LilChucky2001** [crouvpony47/furdown#35]
I did notice i had to authorize my connection to FA this morning when booting the site up, maybe thats stopping the bot?

Errors from log:

"Error: Response status code does not indicate success: 403 (Forbidden)."
"If it is caused by Clouflare validation, pass it and navigate to the FA main page."
"Does not seem to be authorized (or need to pass CF validation) . . ."

**Comment by Deer-Spangle** [comment]
Yeah, that's cloudflare anti-bot protection. Probably FA is being hit by some DDoS attacks in recent days.
It went down for another 12 hours today, but seems to be back up again now.

**Comment by crouvpony47** [comment]
Closing due to a major rework of the authorization mechanism. Please open a new issue for problems that still persist in v.2.0.1.0+.

_After a CF human verification is passed, you sometimes need to also navigate to the front page of the site for it to be detected properly._
